spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/TraceKeys.java

```diff
@@ -318,6 +318,21 @@ public void setHeaders(Collection<String> headers) {
 		 */
 		private String commandKey = "commandKey";
 
+		/**
+		 * Name of the command group
+		 */
+		private String commandGroup = "commandGroup";
+
+		/**
+		 * Name of the cache key
+		 */
+		private String cacheKey = "cacheKey";
```

Comment:
trick with tags is adding the minimum, so that it doesn't expand the spans with unused data. do you expect people to look up by cache key or thread pool key? What if we didn't define these now? (ex instead wait for someone to ask for them). If you look at the http keys, you'll notice documentation about why the keys are important.. prefer less, better documented keys. Ex. command key is used for X, ex you can re-use this in a turbine query like so, to investigate further.